lib/Serialization/Serialization.cpp

```diff
@@ -3235,6 +3250,8 @@ void Serializer::writeDecl(const Decl *D) {
     if (auto errorConvention = fn->getForeignErrorConvention())
       writeForeignErrorConvention(*errorConvention);
 
+    writeInlinableBodyText(fn);
```

Comment:
David would probably say to add "maybe…" or "…IfNeeded" to the name of this method, since it doesn't always do anything.